pkg/sql/backfill.go

```diff
@@ -1217,7 +1225,7 @@ func (sc *SchemaChanger) distIndexBackfill(
 
 	// Setup periodic job details update.
 	stopJobDetailsUpdate := make(chan struct{})
-	detailsDuration := 10 * time.Second
+	detailsDuration := backfillCheckpointInterval.Get(&sc.settings.SV)
```

Comment:
We also update the progress at a 10-second interval, this presumably doesn't cause an issue since it is only updating the fraction progressed. Just making sure we're okay with that.

Reply:
Yeah, I targeted this change at just the large CF. I don't think that we expect the fraction progressed to run into the same issue since the size is contained. Can address in a follow up if we feel differently though.